server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -663,19 +678,26 @@ UploadedMetadataResults writeMetadataInParallel(
         });
         indicesRoutingToUpload.forEach(indexRoutingTable -> {
             uploadTasks.put(
-                InternalRemoteRoutingTableService.INDEX_ROUTING_METADATA_PREFIX + indexRoutingTable.getIndex().getName(),
+                InternalRemoteRoutingTableService.INDEX_ROUTING_METADATA_PREFIX,
```

Comment:
why is this required, seems incorrect?

Reply:
Thanks for calling it out. It was missed during re-based.